### PR TITLE
label stable branch packaging PRs as such

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: prprocessor/__main__.py [hunk 1]
```diff
@@ -395,6 +396,8 @@ async def on_pr_modified(*, action: str, pull_request: Mapping, **_kw) -> None:
         labels.add(Label.DEB)
     elif target_branch.startswith('rpm/'):
         labels.add(Label.RPM)
+    if STABLE_PACKAGING_BRANCH_REGEX.match(target_branch):
```
Comment (evgeni) [inline prprocessor/__main__.py:399]: could also be added in L392, wasn't sure what's more readable.

Comment (ekohl) [reply prprocessor/__main__.py:399]: I was thinking about moving it to the config. We already have it elsewhere. Any thoughts on that?

Comment (evgeni) [reply prprocessor/__main__.py:399]: moving what exactly?

Comment (ekohl) [reply prprocessor/__main__.py:399]: Initially I thought about `stable_branch_prefix` and `stable_branch_suffix` in `config.yaml` but ended up with https://github.com/theforeman/prprocessor/pull/187 which should also fix "Fixed in Releases" for packaging PRs.

**Comment by evgeni** [comment]
#187 was better and merged, closing

**Pull request closed without merging**